src/functions/template-tags/loop.php

```diff
@@ -256,12 +256,13 @@ function tribe_get_upcoming_link() {
 	 * @return bool
 	 */
 	function tribe_has_previous_event() {
+		$wp_query = tribe_get_global_query_object();
+		$has_previous = false;
 
-		if ( ! $wp_query = tribe_get_global_query_object() ) {
-			return;
+		if ( null === $wp_query ) {
+			return apply_filters( 'tribe_has_previous_event', $has_previous );
```

Comment:
Docblocks

Reply:
This is not part of a new filter but is the same filter on the function just in case the where `wp_query` is null the original filter is applied.

Same case for the other one let me know if you think it requires a doccblock as well.